miner/worker.go

```diff
@@ -50,7 +50,7 @@ import (
 )
 
 const (
-	targetTxsSize = 232 * units.KiB
+	targetTxsSize = 1800 * units.KiB
```

Comment:
what is the reason to use 1800 vs 1792 in coreth?

Reply:
There are no atomic transactions here, so I thought it made sense to add a little more to the soft cap.